python/ray/serve/_private/http_proxy.py

```diff
@@ -295,6 +298,19 @@ def get_handle(name):
                 "status_code",
             ),
         )
+
+        self.num_ongoing_requests_gauge = metrics.Gauge(
+            name="serve_num_ongoing_requests",
+            description="The number of ongoing requests in this HTTP Proxy.",
+            tag_keys=("actor_name", "node_id", "node_ip_address"),
+        ).set_default_tags(
+            {
+                "actor_name": actor_name,
```

Comment:
I think actor name is totally redundant with `node_id`

Reply:
Good call, I removed it.